code/game/machinery/syndicatebomb.dm

```diff
@@ -353,4 +354,8 @@
 		message_admins(adminlog)
 		log_game(adminlog)
 	explosion(get_turf(src),20,20,20, flame_range = 20)
-	qdel(src)
\ No newline at end of file
+	qdel(src)
+
+/obj/machinery/blastco_antiburglar/HasProximity(atom/movable/AM as mob|obj)
```

Comment:
why would it? it only detects if you're within a tile from it, like a portable flasher or a proximity grenade